cpp/src/groupby/sort/aggregate.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include <groupby/common/utils.hpp>
```

Comment:
Please include the prefix `<cudf/...>`.

Reply:
`cudf` can't be prepended. These files are in `src/`.

Reply:
What? You have a hpp file in `src`?

Reply:
Yes. There are ~30 hpp files in `src/`. They need not be exposed as public headers. So they remain in `src/`.